regression/cbmc/Makefile

```diff
@@ -4,7 +4,7 @@ test:
 	@../test.pl -p -c "../../../src/cbmc/cbmc --validate-goto-model" -X smt-backend
 
 test-cprover-smt2:
-	@../test.pl -p -c "../../../src/cbmc/cbmc --cprover-smt2"
+	@../test.pl -p -c "../../../src/cbmc/cbmc --cprover-smt2" -X broken-smt-backend
```

Comment:
The commits are in the wrong order, this should go after the broken-smt-backend tag is introduced

Reply:
I think that's just GitHub displaying them in the wrong order, but let me make this unambiguous by moving this particular change into the same commit that introduces the tag.

Reply:
You can update the commit date to make github display them in the right order `git rebase "$(git merge-base HEAD origin/develop)" --ignore-date -x 'git commit --amend -C HEAD --date="$(date -R)" && sleep 1.05' -i`